### PR TITLE
ADD first version.

### PR DESCRIPTION
**Pull request by labajo**
First version of fiware-here-adaptor

### CODE REVIEW

File: cbProvisioner/index.js [hunk 1]
```diff
@@ -0,0 +1,193 @@
+'use strict';
+
+var Orion = require('fiware-orion-client');
+var fs = require('fs');
+//var ORION_SERVER = 'http://130.206.121.52:1026/v1';
+var ORION_SERVER = 'http://130.206.83.68:1026/v1';
```
Comment (labajo) [inline cbProvisioner/index.js:5]: Is it convenient to hide this ip address?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,237 @@
-# fiware-here-backend
-Backend implementation of the integration between FIWARE and HERE Maps
+# Fiware HERE Adaptor
+
+Join the information provided by HERE and City Broker for the car navigator.
```
Comment (jmcanterafonseca) [inline README.md]: Adaptor component for the FIWARE-NGSI-HERE integration for Smarter car navigation through cities.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,237 @@
...
+
+# How to install and run
+
+Following instructions are for development setups:
```
Comment (jmcanterafonseca) [inline README.md]: The following

File: README.md [hunk 4]
```diff
@@ -1,2 +1,237 @@
...
+    "geocodingUrl": "http://geocoder.cit.api.here.com",
+    "reverseGeocodingUrl": "http://reverse.geocoder.cit.api.here.com",
+    // You can obtain your API keys in HERE service.
+    "appId": "myAppId",
+    "appCode": "myAppCode"
+  }
+}
+```
+The internal config is in 'lib/config/config.json' but you can use the environment variable CONFIG_DIR to set a folder that merges the configs obtained from json files.
+
```
Comment (jmcanterafonseca) [inline README.md:124]: I cannot understand this statement

File: cbProvisioner/index.js [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+  }
+} else {
+  process.argv.forEach(function (val, index) {
+    if (index > 1) {
+      cities[val] = true;
+    }
+  });
+}
+
+if (cities.madrid) {
```
Comment (jmcanterafonseca) [inline cbProvisioner/index.js]: better with a switch block, right?

File: package.json [hunk 6]
```diff
@@ -0,0 +1,67 @@
+{
+  "name": "fiware-here-adaptor",
+  "version": "1.0.0",
+  "description": "Join the information provided by HERE and City Broker for the car navigator.",
```
Comment (jmcanterafonseca) [inline package.json]: align description with what was proposed for the README

**Comment by jmcanterafonseca** [comment]
@labajo thanks for the work. looks good with the minor suggested amendments

File: cbProvisioner/index.js (hunk 5 above)
Comment (labajo) [inline cbProvisioner/index.js]: A switch without breaks statements inside its cases.

**Pull request merged**